examples/termination/conda.yaml

```diff
@@ -0,0 +1,10 @@
+# https://github.com/robocorp/rcc/blob/master/docs/recipes.md#what-is-in-condayaml
+
+channels:
+  - conda-forge
+
+dependencies:
+  - python=3.9.12               # Install obscure python version
```

Comment:
It doesn't matter too much, but I would suggest to go with the latest versions here

Reply:
Done